packages/react-magma-dom/src/components/Table/Table.stories.tsx

```diff
@@ -51,7 +52,7 @@ const rows = [
 
 const Template: Story<TableProps> = args => (
   <Card isInverse={args.isInverse}>
-    <Table {...args}>
+    <Table tableTitle="Basic usage" {...args}>
```

Comment:
This isn't needed as it will always get overwritten by `args` as long as the Default has tableTitle